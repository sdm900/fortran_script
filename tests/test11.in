do i=1,10
print i," before cycle"
if(i>5) then
exit
endif
print i," after cycle"
enddo
print i," after do loop"

