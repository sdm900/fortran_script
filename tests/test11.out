     1.00000 before cycle 
     1.00000 after cycle 
     2.00000 before cycle 
     2.00000 after cycle 
     3.00000 before cycle 
     3.00000 after cycle 
     4.00000 before cycle 
     4.00000 after cycle 
     5.00000 before cycle 
     5.00000 after cycle 
     6.00000 before cycle 
     6.00000 after do loop 
